src/containers/Tenant/Diagnostics/Consumers/Consumers.tsx

```diff
@@ -17,18 +19,32 @@ const b = block('ydb-consumers');
 
 interface ConsumersProps {
     path: string;
+    pathType?: EPathType;
+    schemaNestedChildrenPaths?: string[];
 }
 
-export const Consumers = ({path}: ConsumersProps) => {
+export const Consumers = ({path, pathType, schemaNestedChildrenPaths}: ConsumersProps) => {
     const dispath = useDispatch();
 
+    const isCdcStream = pathType === EPathType.EPathTypeCdcStream;
+
+    const targetPath = useMemo(
+        () =>
+            // Since every CDC Stream have only one nested PersQueueGroup, we select only first item
+            (isCdcStream ? schemaNestedChildrenPaths && schemaNestedChildrenPaths[0] : path) ||
+            path,
+        [path, pathType, schemaNestedChildrenPaths],
+    );
+
     const fetchData = () => {
-        dispath(getDescribe({path}));
+        dispath(getDescribe({path: targetPath}));
     };
 
-    useAutofetcher(fetchData, [path]);
+    useAutofetcher(fetchData, [path, pathType, schemaNestedChildrenPaths]);
```

Comment:
fetchData does not depend from pathType and schemaNestedChildrenPaths, why are they here?